config/component_macros.cmake

```diff
@@ -319,10 +319,6 @@ macro( add_component_library )
     OUTPUT_NAME ${acl_LIBRARY_NAME_PREFIX}${acl_LIBRARY_NAME}
     FOLDER      ${folder_name}
     WINDOWS_EXPORT_ALL_SYMBOLS ON )
- if("${acl_LINK_LANGUAGE}" STREQUAL "CUDA")
-   set_property( TARGET ${acl_TARGET} APPEND PROPERTY
-     COMPILE_DEFINITIONS "USE_CUDA=ON" )
```

Comment:
Did you do a global search for `USE_CUDA` to ensure this doesn't disable anything? I'm assuming that you did, but I need to ask the question.

Reply:
I can see where `USE_CUDA` is in code and tried to make sure this didn't disable anything. This is where the `COMPILER_DEFINITION` is a littler weird to me--there could be code in Jayenne that does not see the definition of `USE_CUDA` in a `config` header but was previously doing the right thing because `USE_CUDA` was previously set with these `COMPILE_DEFINITIONS` commands. I would expect compilation to fail if the wrong fork of a preprocessor macro was chosen.

Maybe it makes more sense to use the `COMPILE_DEFINITIONS` but use it on code and not just CUDA?